Reply/app/src/main/java/com/materialstudies/reply/ui/home/EmailViewHolder.kt

```diff
@@ -44,7 +43,21 @@ class EmailViewHolder(
     init {
         binding.run {
             this.listener = listener
-            attachmentRecyclerView.adapter = attachmentAdapter
+            composeView.apply {
+                setViewCompositionStrategy(
+                    ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed
```

Comment:
Nit: since ComposeView is used in a ViewHolder, ideally this should use the default strategy so it follows our guidance

Reply:
Removed, thanks!